newm-server/docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3.9"
+services:
+  newm-server:
+    depends_on:
+      - postgres
+    image: newm-server:latest
+    ports:
+      - "3939:3939"
+    environment:
+      - ARWEAVE_WALLET_JSON=${ARWEAVE_WALLET_JSON}
+      - AWS_AGREEMENT_BUCKET=${AWS_AGREEMENT_BUCKET}
+      - AWS_AUDIO_BUCKET=${AWS_AUDIO_BUCKET}
+      - AWS_AUDIO_CLOUDFRONT_HOST_URL=${AWS_AUDIO_CLOUDFRONT_HOST_URL}
+      - AWS_AUDIO_SQS_QUEUE_URL=${AWS_AUDIO_SQS_QUEUE_URL}
+      - AWS_MINTING_SQS_QUEUE_URL=${AWS_MINTING_SQS_QUEUE_URL}
+      - AWS_PROFILE=${AWS_PROFILE}
+      - AWS_REGION=${AWS_REGION}
+      - REGION=${AWS_REGION}
+      - AWS_SECRET_ARN=${AWS_SECRET_ARN}
+      - CLOUDINARY_URL=${CLOUDINARY_URL}
+      - CORS_HOSTS=${CORS_HOSTS}
+      - EMAIL_SMTP_PORT=465
+      - EVEARA_SERVER=${EVEARA_SERVER}
+      - NEWM_CHAIN_HOST=${NEWM_CHAIN_HOST}
```

Comment:
I chose to keep newm-chain out of the compose file for now - although it can be added when its needed